_maps/shuttles/whiteship_meta.dmm

```diff
@@ -989,9 +989,9 @@
 	},
 /obj/item/clothing/gloves/color/black,
 /obj/item/clothing/gloves/color/black,
-/obj/item/clothing/suit/armor/vest,
-/obj/item/clothing/suit/armor/vest,
 /obj/structure/table,
+/obj/item/clothing/suit/armor/swat,
+/obj/item/clothing/suit/armor/swat,
```

Comment:
not really? I changed it so the armor that was to go with the SWAT helm had the adequate vest armor